example/src/main.js

```diff
@@ -0,0 +1,32 @@
+import Grid from '../../src/Grid/Grid';
+import React from 'react';
+
+const columns = [
+  { 
+    name: 'key', 
+    label: 'label', 
+    sortable: true,
+    sortDirection: 'none',
+    searchable: true,
+    visible: true,
```

Comment:
Don't change this now, but add a ticket to allow to have some default values like visible is always true. Check original tubular for default values.

Reply:
We can set visible as a non required property and then verify if visible is present if it is not then we take true as default

Reply:
Sure, add a ticket for that. Merging...